src/bessel.jl

```diff
@@ -499,7 +509,7 @@ Bessel function of the first kind of order 0, ``J_0(x)``.
 """
 function besselj0(x::BigFloat)
     z = BigFloat()
-    ccall((:mpfr_j0, :libmpfr), Int32, (Ptr{BigFloat}, Ptr{BigFloat}, Int32), &z, &x, ROUNDING_MODE[])
+    ccall((:mpfr_j0, :libmpfr), Int32, (Ref{BigFloat}, Ref{BigFloat}, Int32), z, x, ROUNDING_MODE[])
```

Comment:
Correct. It always pass an pointer to a memory with the same layout as the object being `Ref`d (don't know how to describe this but hopefuly that's unambiguous). For `!isbits` objects this is the object itself and for a copy for `isbits` ones.